contracts/test/Gateway.t.sol

```diff
@@ -858,13 +861,25 @@ contract GatewayTest is Test {
                 SetTokenTransferFeesParams({
                     assetHubCreateAssetFee: createTokenFee * 2,
                     registerTokenFee: registerTokenFee,
-                    assetHubReserveTransferFee: sendTokenFee
+                    assetHubReserveTransferFee: sendTokenFee * 3,
+                    destinationMaxTransferFee: maxDestinationFee + 1
                 })
             )
         );
         fee = IGateway(address(gateway)).quoteRegisterTokenFee();
         // since deliveryCost not changed, so the total fee increased only by 50%
         assertEq(fee, 7500000000000000);
+        // register token first
+        IGateway(address(gateway)).registerToken{value: fee}(address(token));
+
+        fee = IGateway(address(gateway)).quoteSendTokenFee(address(token), assetHubParaID, 0);
+        // Fee should increase
+        assertEq(fee, 10000000000000000);
+
+        ParaID destPara = ParaID.wrap(2043);
+        fee = IGateway(address(gateway)).quoteSendTokenFee(address(token), destPara, 1e11 + 1);
+        // Max fee should increase
```

Comment:
Why not use the `maxDestinationFee` param here?

Reply:
addressed in [ddbdd58](https://github.com/Snowfork/snowbridge/pull/1168/commits/ddbdd58ac013440a585ed01f30a5dfad3f091e11)